### PR TITLE
Adding platform timeout configuration

### PR DESCRIPTION
**Pull request by Albertoimpl**

### CODE REVIEW

File: spring-cloud-app-broker-docs/src/docs/asciidoc/deployment-platforms.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ spring:
           *client-secret: EXAMPLE_SECRET*
           *default-org: test*
           *default-space: development*
+          *staging-timeout: 15*
+          *startup-timeout: 10*
```
Comment (kvmw) [inline spring-cloud-app-broker-docs/src/docs/asciidoc/deployment-platforms.adoc]: I believe this was supposed to be `deployment-timeout`.

**Review by kvmw — APPROVED** [review]
I verified the snapshot build and is working  as expected.  Thank you!

The only minor suggestion I have is a message for the failure, If possible. 

Currently, the SI status is shown like the following, in case of timeout error:
```
Showing status of last operation:
   status:    update failed
   message:
   started:   2024-09-26T14:04:45Z
   updated:   2024-09-26T14:04:45Z
```
Maybe having a message like "timeout waiting for deployment" or such can help users without the need to check the logs.

File: spring-cloud-app-broker-docs/src/docs/asciidoc/deployment-platforms.adoc (hunk 1 above)
Comment (Albertoimpl) [reply spring-cloud-app-broker-docs/src/docs/asciidoc/deployment-platforms.adoc]: Good catch, thanks!

**Comment by Albertoimpl** [comment]
Thanks for testing it @kvmw! I like your suggestion regarding error messages but it would require a different set of changes. Mind filing a new issue?

**Pull request merged**